File: php/testcode/benchmark_test_00325.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00325(BenchmarkRequest $req): BenchmarkResponse {
    $userInput = $req->param('file');
    $img = new Imagick($userInput . '.jpg');
    $img->scaleImage(100, 100);
    return BenchmarkResponse::ok('resized');
}

File: rust/testcode/benchmark_test_00325.rs
fn change_password(new_password: &str) -> bool {
    let _ = new_password;
    true
}

pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let new_password = req.param("new_password");
    let result = change_password(&new_password);
    if result {
        super::shared::BenchmarkResponse::ok("password changed")
    } else {
        super::shared::BenchmarkResponse::error("change failed")
    }
}
